chapters/samplers.txt

```diff
@@ -351,7 +356,14 @@ ifdef::VK_EXT_fragment_density_map[]
     specifies that the implementation may: use approximations when
     reconstructing a full color value for texture access from a subsampled
     image.
+endif::VK_EXT_fragment_density_map[]
+ifdef::VK_EXT_non_seamless_cube_map[]
+  * ename:VK_SAMPLER_CREATE_NON_SEAMLESS_CUBE_MAP_BIT_EXT
+    specifies that <<textures-cubemapedge, cube map edge handling>> is not
```

Comment:
I think this needs to be spelled out in the edge handling section too; right now it blanket says that edge handling happens for cubemaps.

Reply:
I'm not sure I understand where you want to have it spelled out. I already added a section to say that border texel replacement happens for cube maps if this bit is used. So the "Otherwise the texel coordinates lie beyond the edges or corners of the selected cube map face, and Cube map edge handling is performed." doesn't apply anymore.

Reply:
You're not wrong - what's tripping me up as a reader is that the second sentence in that section calls out an exception - and this is another one. Reading that section alone it's thus not clear. I think changing this:

> If the texel coordinates lie beyond the edges or corners of the selected cube map face, the following steps are performed. Note that this does not occur when using VK_FILTER_NEAREST filtering within a mip level, since VK_FILTER_NEAREST is treated as using VK_SAMPLER_ADDRESS_MODE_CLAMP_TO_EDGE.

to

> If the texel coordinates lie beyond the edges or corners of the selected cube map face (as described in the prior section).

Then I think it makes it substantially clearer.

A bigger rewrite is probably warranted at some point, but I don't think it'd be appropriate to ask you to do that in this PR 😬

Reply:
Okay, I added that part. But I kept the note about nearest filtering, I'm not sure why that should be removed.

Reply:
The problem I have with the nearest filtering note is that it's calling out one exception when really there's a bunch of them. This often trips readers up, as they "read between the lines" by assuming that it's the only exception.

However, IMO that little addendum fixes it well enough for this extension, so I'm happy to call this resolved.